spec/tbd_tests_spec.rb

```diff
@@ -2507,23 +4351,23 @@
     # closer to details of another Renzo Piano extension: the Modern Wing of the
     # Art Institute of Chicago. Both galleries are similar in that daylighting
     # is zenithal, covering all (or nearly all) of the roof surface. In the
-    # case of Chicago, the roof is entirely glazed (as reflected in the model):
+    # case of Chicago, the roof is ~entirely glazed (as reflected in the model).
     #
     # www.archdaily.com/24652/the-modern-wing-renzo-piano/
     # 5010473228ba0d42220015f8-the-modern-wing-renzo-piano-image?next_project=no
     #
-    # This is allowed in OpenStudio, yet SketchUp (via the plugin) would
-    # complain.
+    # However, a small 1" strip is maintained along the South roof/wall edge of
+    # the gallery to ensure skylight area < roof area.
```

Comment:
Fixing up the model for forward translation. Adding a 1" strip to offset full-size skylight. The SDK allows subsurface area == base surface area, but EnergyPlus doesn't.